tests/jq.test

```diff
@@ -1848,6 +1848,26 @@ map(abs)
 [0.1,1000000000000000002]
 [1e-1, 1000000000000000002]
 
+nwise(2)
+null
+null
+
+nwise(2)
+[]
+[]
```

Comment:
Add a negative $n test case as it seems to get stuck atm, should throw error?

Reply:
Good points.  QED. Tx.